src/GithubClientFactory.ts

```diff
@@ -0,0 +1,41 @@
+import {GithubClientConfigs} from './Config';
+import {Logger} from "./Logger";
+/* const { throttlingPlugin } = require("@octokit/plugin-throttling");
+const { retryPlugin } = require("@octokit/plugin-retry");
+
+ */
+const Octokit = require("@octokit/rest").plugin(require("@octokit/plugin-throttling")).plugin(require("@octokit/plugin-retry"));
+
+export class GithubClientFactory  {
+    octokitClient;
```

Comment:
Any implicite. On a déjà accès au typage de Octokit!

```suggestion
    private readonly octokitClient: Octokit;

```